src/_igraph/convert.c

```diff
@@ -2538,11 +2606,12 @@ int igraphmodule_PyObject_to_vector_ptr_t(PyObject* list, igraph_vector_ptr_t* v
     if (igraph_vector_ptr_push_back(vec, subvec)) {
       Py_DECREF(it);
       igraph_vector_destroy(subvec);
+      free(subvec);
       igraph_vector_ptr_destroy_all(vec);
       return 1;
     }
 
-    /* ownership of 'subvec' taken by 'vec' here */
+    /* ownership of 'subvec' taken by 'vec' here, we should still free the pointer */
```

Comment:
Here there is no need to free the pointer.

The code seems correct to me. I'm just commenting your change to the comment.

Reply:
Indeed, the code is correct, the comment is not.

Reply:
ok thanks